docs/basics/functions.md

```diff
@@ -25,6 +25,8 @@ Flex has many useful functions, which can be combined in different ways to help
     -   [to_lower](#to_lower)
     -   [value_parser](#value_parser)
     -   [value_transformer](#value_transformer)
+    -   [sample_include_filter](#sample_include_filter)
```

Comment:
@haihongren Please move the functions in the list up so that alphabetical order is respected.